server/src/utils/email.util.ts

```diff
@@ -0,0 +1,27 @@
+import transporter from "../connections/email.connect";
+import {AppError} from "../utils/app-error.util";
+import { EMAIL_USER } from "./environment-variables";
+
+interface IEmailOptions {
+    to: string;
+    text: string;
+    html?: string;
+}
+
+const sendEmail = async (options: IEmailOptions): Promise<void> => {
+    const { to, text, html } = options;
+    try {
+        const result = await transporter.sendMail({
+            from: EMAIL_USER,
+            to,
+            subject: "Cengineer3",
+            text,
+            html,
+        });
+        console.log(`Email sent: ${result.response}`);
+    } catch (error) {
+        throw new AppError("Email sending failed", 500,error.message, true);
+    }
+};
```

Comment:
subject should be generic as well and not const string "Cengineer3"

Reply:
later you need to fix AppError from bugfix branch.
because we're using it differently now

Reply:
why html it's optional ?